src/extension.ts

```diff
@@ -34,10 +34,10 @@ export async function activate(context: vscode.ExtensionContext) {
   };
 
   const syncWorkspaceFolders = () => {
-    if (extensions.value == null || extensions.value.length == 0) {
-      const msg = "node: test runner:  nodejs-testing.extensions array is empty. Please remove the setting 'nodejs-testing.extensions' or define at least one element.";
+    if (extensions.value?.length==0) {
```

Comment:
Please leave it as this, as a value of null will now not trigger this check (`null != 0`). Also, I strictly avoid use of `==` and prefer `===` 😛 

```suggestion
    if (extensions.value?.length) {
```

Reply:
Oops, that's a bug. I've checked in your first suggestion: !extensions.value?.length.

Unittest:

![image](https://github.com/connor4312/nodejs-testing/assets/67872991/11fc9892-738b-4b9e-ba77-87e717eac8a1)